modules/rosa-sts-creating-a-cluster-quickly.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started_sts/rosa_creating_a_cluster_with_sts/rosa-sts-creating-a-cluster-quickly.adoc
+
+[id="rosa-sts-creating-cluster-using-defaults{context}"]
+= Creating a ROSA cluster with STS using the default options
+
+Through the {product-title} CLI (`rosa`), you can quickly create an OpenShift cluster that uses the AWS Security Token Service (STS). The `--sts` argument enables you to deploy a cluster with STS using the default installation options.
+
```

Comment:
I'd remove this last sentence relating to the `--sts` flag since it is not clear where that flag gets used and what default options it applies to.

Reply:
@0kashi thank you for your detailed review and for this suggestion. I will remove this line.